api/index.js

```diff
@@ -1 +1,30 @@
-module.exports = { v1: require('./v1') };
+var config = require('./config');
+var logger = require('./logging');
+
+var v1 = require('./v1');
+
+logger.info("starting superuser setup check in the background");
+
+// the following uses the V1 API to ensure that a superuser is present on the
+// currently-running instance. we might want to consider a way to do this without
+// using version-specific functionality
```

Comment:
It's using the `v1` API functionality to do this. I just was thinking that it'd be kind of weird to have part of the initialization process for the entire API (which might include more versions than just `v1` in the future) importing code from a specific version like that. 

Might not wind up being that strange, but just something I wanted to consider if/when we do have more versions.